packages/axios-extension/test/abap/ui5-abap-repository-service.test.ts

```diff
@@ -172,6 +173,9 @@ describe('Ui5AbapRepositoryService', () => {
 
     describe('createPayload', () => {
         test('ensure special characters are encoded', async () => {
+            /**
+             *
+             */
```

Comment:
?

Reply:
Yeah this is an oddity, it causes an error to remove it.  
Would need to disable jsdoc completely for test files to remove it. 
Maybe it is the best option anyway?

Reply:
https://github.com/SAP/open-ux-tools/pull/689/commits/ced70fe6223b7c74af5523aa11f9aab255950af6